# HG changeset patch
# User Arlo Breault <arlolra@gmail.com>
# Date 1472485829 25200
#      Mon Aug 29 08:50:29 2016 -0700
# Branch THUNDERBIRD4530_2016082513_RELBRANCH
# Node ID a13735e63c2e5c6209fefc091263454079a5e517
# Parent  69baf6e1ea1e4c8f4ddf719bff6b542869a99a23
Remove backwards compat. normalize in twitter prpl

 * Just use .toLowerCase()

diff --git a/chat/protocols/twitter/twitter.js b/chat/protocols/twitter/twitter.js
--- a/chat/protocols/twitter/twitter.js
+++ b/chat/protocols/twitter/twitter.js
@@ -384,21 +384,16 @@ function Account(aProtocol, aImAccount)
   this._init(aProtocol, aImAccount);
   this._knownMessageIds = new Set();
   this._userInfo = new Map();
   this._friends = new Set();
 }
 Account.prototype = {
   __proto__: GenericAccountPrototype,
 
-  // The correct normalization for twitter would be just toLowerCase().
-  // Unfortunately, for backwards compatibility we retain this normalization,
-  // which can cause edge cases for usernames with underscores.
-  normalize: aString => aString.replace(/[^a-z0-9]/gi, "").toLowerCase(),
-
   consumerKey: Services.prefs.getCharPref("chat.twitter.consumerKey"),
   consumerSecret: Services.prefs.getCharPref("chat.twitter.consumerSecret"),
   completionURI: "http://oauthcallback.local/",
   baseURI: "https://api.twitter.com/",
   _lastMsgId: "",
 
   // Use this to keep track of the pending timeline requests. We attempt to fetch
   // home_timeline, @ mentions and tracked keywords (i.e. 3 timelines)
